README.md

```diff
@@ -181,49 +184,85 @@ terminal or deactivate the Sherpa environment.
 Source Build
 ------------
 
-
-### 2. Build from source tarball
-
-If you donwloaded the Sherpa source tarball, you can build it by:
-
-    $ tar xf sherpa-<version>.tar.gz
-    $ cd sherpa-<version>
-    $ python setup.py install
-
-Prerequisites:
+The prerequisites for building from source are:
 
  - Python: `setuptools`, `numpy`
  - System: `gcc`, `g++`, `gfortran`, `make`, `flex`, `bison`
 
+At present it is suggested that Sherpa be built with Python 2.7.
+
+It is *highly* recommended that [`matplotlib`](http://matplotlib.org/)
+be installed, as this is used to create graphical output (although the
+code can be built and used without this package), and
+[`ipython`](http://ipython.org/), which is for interactive analysis.
+Data I/O requires the [`pyfits`](http://www.stsci.edu/institute/software_hardware/pyfits) 
+package.
+
 The instructions on how to set up the prerequisites vary from system to system,
 and even on the same system there may be multiple ways of setting up the requirements.
 
-To test that your installation of Sherpa is working, type:
+There are two ways to download the source code:
 
-    $ sherpa_test
+ - as a source tarball
+
+ - directly from GitHub as a git repository
+
+### 2a. Extract the source tarball
+
+If you donwloaded the Sherpa source tarball, you can extract it by:
+
+    $ tar xf sherpa-<version>.tar.gz
+    $ cd sherpa-<version>
 
-### 3. Get the code from the GitHub repository
+### 2b. Get the code from the GitHub repository
 
 You can clone the Sherpa repository with:
 
     $ git clone https://github.com/sherpa/sherpa
+    $ cd sherpa
 
 The most stable code is available in the `stable` branch. The main development
 code, which is unstable, is available in the `master` branch. New features
 and bug fixes or other, even less
 stable versions of the code may be available in other branches.
 
+### 2c. Build Sherpa
+
+Once the code is available, it can be built and installed with:
+
+    $ python setup.py install
+
+### 2d. Testing the build
+
+To test that your installation of Sherpa is working, type:
+
+    $ sherpa_test
+
+which will run a small test suite (the location of this script depends
+on where the installation step chose). Note that the test may report
```

Comment:
"the location of this script depends on where the installation step chose" -> It sounds like something is missing.